sdk/tables/Azure.Data.Tables/src/TableServiceClient.cs

```diff
@@ -15,15 +16,43 @@ public class TableServiceClient
         private readonly TableInternalClient _tableOperations;
         private readonly OdataMetadataFormat _format = OdataMetadataFormat.ApplicationJsonOdataFullmetadata;
 
-        public TableServiceClient(Uri endpoint, TablesSharedKeyCredential credential, TableClientOptions options = null)
+        public TableServiceClient(Uri endpoint)
+                : this(endpoint, default(TableSharedKeyPipelinePolicy), null) { }
```

Comment:
Nit - maybe call through to the `TableServiceClient(endpoint, options)` instead of the maximal overload.  It makes it less painful when you have to add a new parameter to the maximal overload.